randr.c

```diff
@@ -21,15 +21,51 @@
  */
 
 #include "xplugd.h"
+#include "edid.h"
 
 static char *con_actions[] = { "connected", "disconnected", "unknown" };
 
+int get_monitor_name(char *name, char *monitor_name)
```

Comment:
Another, semi-serious, issue is this: `char *name` should probably be `const char *name` and the pointer to the resulting string, `monitor_name`, should really have an accompanying extra argument `size_t len` instead of hard-coding 14 in the `strncpy()`.